src/jdk.crypto.cryptoki/share/native/libj2pkcs11/p11_util.c

```diff
@@ -182,35 +182,56 @@ void removeAllModuleEntries(JNIEnv *env) {
 /*
  * function to convert a PKCS#11 return value into a PKCS#11Exception
  *
- * This function generates a PKCS#11Exception with the returnValue as the errorcode
- * if the returnValue is not CKR_OK. The functin returns 0, if the returnValue is
- * CKR_OK. Otherwise, it returns the returnValue as a jLong.
+ * This function generates a PKCS#11Exception with the returnValue as the
+ * errorcode. If the returnValue is not CKR_OK. The functin returns 0, if the
+ * returnValue is CKR_OK. Otherwise, it returns the returnValue as a jLong.
  *
  * @param env - used to call JNI funktions and to get the Exception class
```

Comment:
NitPick: here and above some German seems to have slipped in. I think we want "functions"

Reply:
I didn't notice the German, just copy-n-paste. Will fix the typos.

Reply:
Other PKCS#11 JNI sources have the same German typos, I prefer to leave them out of this. Will fix them when they are touched in other fixes.